WcaOnRails/app/models/continent.rb

```diff
@@ -11,11 +11,11 @@ def self.country_ids(continent_id)
   end
 
   def name
-    I18n.t(recordName, scope: :continents)
+    I18n.t(read_attribute(:name), scope: :continents)
```

Comment:
Ok! I agree that spaces inside of YAML key names are a bit weird. I just checked, and these would be our very first such keys. Another option (which I'd be supportive of) would be to introduce a human readable "slug" for each continent, and instead use that for looking up translations. The current id field and name fields are useless, because we use yaml for human readable text now.

@jonatanklosko, do you think spaces in keys would be a problem for your translation app?